consumatio/external/api.py

```diff
@@ -48,6 +51,8 @@ def resolve_movie(*_, code: int, country: str) -> dict:
     :param country: <str> Country abbreviation to get corresponding providers (e.g. "DE" -> Germany)
     :return: <dict> Details of the movie
     """
+    logger.info("Movie was queried -> code:{}, country:'{}'".format(
```

Comment:
This arrow („->“) notation in the logging messages looks quite unusual. Is this any convention? Maybe we should look at the layout of existing logging messages and adapt to their style.